code/orders-api-rest/api/openapi-rest.yml

```
@@ -2,6 +2,7 @@
 info:
   title: Retail API
   description: Retail service API
+  version: 2.4.0
   version: 2.3.0
```

YAMLlint (1.37.1)

[error] duplication of key "version" in mapping

(key-duplicates)